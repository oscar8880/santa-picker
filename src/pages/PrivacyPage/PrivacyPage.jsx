import styles from './PrivacyPage.module.scss';
import React from 'react';
import Text from '../../components/Text/Text';
import PageWrapper from '../../components/PageWrapper/PageWrapper';
import { Container, Row, Col } from 'react-grid-system';

const PrivacyPage = () => {
  return (
    <div className={styles.PrivacyPage}>
      <PageWrapper>
        <Container fluid>
          <Row justify="start">
            <Col xs={12} md={8} lg={8}>
              <div className={styles.PrivacyPage__Title}>
                <Text appearance="h1" tagName="h1">
                  Privacy Policy
                </Text>
              </div>
              <div className={styles.PrivacyPage__Text}>
                <Text>Version: September 10, 2020</Text>
              </div>
              <div className={styles.PrivacyPage__Text}>
                <Text>
                  Santa Picker is a personal project created primarly as a
                  learning exercise.
                </Text>
              </div>
              <div className={styles.PrivacyPage__Subtitle}>
                <Text tagName="h2" appearance="h5">
                  How is personal data used?
                </Text>
              </div>
              <div className={styles.PrivacyPage__Text}>
                <Text>
                  A limited amount of personal data is processed solely in order
                  to perform the function of picking secret santa names and
                  informing participants of the results.
                </Text>
              </div>
              <div className={styles.PrivacyPage__Text}>
                <Text>
                  Your data is not provided to third parties such as
                  advertisers.
                </Text>
              </div>
              <div className={styles.PrivacyPage__Subtitle}>
                <Text tagName="h2" appearance="h5">
                  What personal data is stored?
                </Text>
              </div>
              <div className={styles.PrivacyPage__Text}>
                <Text>
                  In order to use the name picking service, you are required to
                  sign up by providing an email address or using a Google
                  account. This email address is stored securely on the Auth0
                  platform and is used solely for the purpose of signing in to
                  the site. You will not be sent emails beyond the emails which
                  deliver name picking results.
                </Text>
              </div>
              <div className={styles.PrivacyPage__Text}>
                <Text>No other personal data is stored.</Text>
              </div>
            </Col>
          </Row>
        </Container>
      </PageWrapper>
    </div>
  );
};

export default PrivacyPage;
