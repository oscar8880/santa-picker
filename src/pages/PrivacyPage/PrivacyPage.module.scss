@import "../../scss/functions";
@import "../../scss/variables";

.PrivacyPage {
  display: flex;
  flex-direction: column;
  padding-bottom: $spacer-12;
  padding-top: $spacer-10;
  @media (min-width: $breakpoint-md) {
    padding-top: $spacer-12;
  }
}

.PrivacyPage__Title {
  padding-bottom: $spacer-8;
  text-align: center;
  @media (min-width: $breakpoint-md) {
    min-width: 940px;
    text-align: left;
  }
}

.PrivacyPage__Subtitle {
  padding-bottom: $spacer-6;
  text-align: center;
  @media (min-width: $breakpoint-md) {
    text-align: left;
  }
}

.PrivacyPage__Text {
  color: $grey-500;
  padding-bottom: $spacer-6;
  text-align: center;
  @media (min-width: $breakpoint-md) {
    text-align: left;
  }
}

.PrivacyPage__CTA {
  display: flex;
  justify-content: center;
  padding-bottom: $spacer-8;

  @media (min-width: $breakpoint-md) {
    justify-content: flex-start;
  }
}
