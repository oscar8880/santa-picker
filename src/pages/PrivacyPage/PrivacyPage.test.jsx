import React from 'react';
import { render } from '@testing-library/react';

import PrivacyPage from './PrivacyPage';

test('matches snapshot', async () => {
  const { asFragment } = render(<PrivacyPage />);

  expect(asFragment()).toMatchSnapshot();
});
