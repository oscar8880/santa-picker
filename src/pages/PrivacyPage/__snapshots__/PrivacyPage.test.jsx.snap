// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="PrivacyPage"
  >
    <div
      class="PageWrapper"
    >
      <div
        style="box-sizing: border-box; position: relative; margin-left: auto; margin-right: auto; padding-left: 15px; padding-right: 15px;"
      >
        <div
          style="margin-left: -15px; margin-right: -15px; display: flex; flex-wrap: wrap; flex-grow: 0; flex-shrink: 0; align-items: normal; justify-content: flex-start;"
        >
          <div
            style="box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 66.66666666666667%; flex-basis: 66.66666666666667%; flex-grow: 0; flex-shrink: 0; max-width: 66.66666666666667%; margin-left: 0%;"
          >
            <div
              class="PrivacyPage__Title"
            >
              <h1
                class="Text Text__h1"
              >
                Privacy Policy
              </h1>
            </div>
            <div
              class="PrivacyPage__Text"
            >
              <p
                class="Text Text__p"
              >
                Version: September 10, 2020
              </p>
            </div>
            <div
              class="PrivacyPage__Text"
            >
              <p
                class="Text Text__p"
              >
                Santa Picker is a personal project created primarly as a learning exercise.
              </p>
            </div>
            <div
              class="PrivacyPage__Subtitle"
            >
              <h2
                class="Text Text__h5"
              >
                How is personal data used?
              </h2>
            </div>
            <div
              class="PrivacyPage__Text"
            >
              <p
                class="Text Text__p"
              >
                A limited amount of personal data is processed solely in order to perform the function of picking secret santa names and informing participants of the results.
              </p>
            </div>
            <div
              class="PrivacyPage__Text"
            >
              <p
                class="Text Text__p"
              >
                Your data is not provided to third parties such as advertisers.
              </p>
            </div>
            <div
              class="PrivacyPage__Subtitle"
            >
              <h2
                class="Text Text__h5"
              >
                What personal data is stored?
              </h2>
            </div>
            <div
              class="PrivacyPage__Text"
            >
              <p
                class="Text Text__p"
              >
                In order to use the name picking service, you are required to sign up by providing an email address or using a Google account. This email address is stored securely on the Auth0 platform and is used solely for the purpose of signing in to the site. You will not be sent emails beyond the emails which deliver name picking results.
              </p>
            </div>
            <div
              class="PrivacyPage__Text"
            >
              <p
                class="Text Text__p"
              >
                No other personal data is stored.
              </p>
            </div>
          </div>
        </div>
        <span
          style="display: table; clear: both;"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
